src/initialization/InitMeshRefinement.cpp

```diff
@@ -28,12 +28,41 @@ namespace impactx
 {
     /** Tag cells for refinement.  TagBoxArray tags is built on level lev grids.
      *
-     * @todo this function is not (yet) implemented.
+     * @TODO handle ngrow argument?
      */
-    void ImpactX::ErrorEst (int lev, amrex::TagBoxArray& tags, amrex::Real time, int ngrow)
+    void ImpactX::ErrorEst (int lev, amrex::TagBoxArray& tags, amrex::Real /* time */, int /* ngrow */)
```

Comment:
@WeiqunZhang: `AmrCore::regrid()` will call this

Example: AMReX https://github.com/AMReX-Codes/amrex/blob/b84d7c069cef7470f195b250926ca0e84ec46fb2/Tests/Amr/Advection_AmrCore/Source/AmrCoreAdv.cpp#L567

Note: we can ignore `time` or pass in something relative to `s`?
Note: we can ignore `ngrow`, too.